ingress/single-cluster/ingress-cloudarmor/README.md

```diff
@@ -81,48 +78,78 @@ Steps:
-
-   ```bash
-   gcloud container clusters get-credentials cluster-test \
-   --zone europe-central2-a
-   ````
+5. Wait until all created objects reach desired state
+```bash
+kubectl describe ingress
+Name:             cloudarmor-test
+Labels:           <none>
+Namespace:        default
+Address:          34.160.135.174
+Ingress Class:    <none>
+Default backend:  <default>
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *           
+              /whereami   whereami:80 (10.24.0.31:8080,10.24.1.47:8080,10.24.2.39:8080)
+Annotations:  ingress.kubernetes.io/backends:
+                {"k8s1-02fed221-default-whereami-80-cda8fb8c":"HEALTHY","k8s1-02fed221-kube-system-default-http-backend-80-37075a2d":"HEALTHY"}
+              ingress.kubernetes.io/forwarding-rule: k8s2-fr-20aeohkx-default-cloudarmor-test-0aa8l6ah
+              ingress.kubernetes.io/target-proxy: k8s2-tp-20aeohkx-default-cloudarmor-test-0aa8l6ah
+              ingress.kubernetes.io/url-map: k8s2-um-20aeohkx-default-cloudarmor-test-0aa8l6ah
+              kubernetes.io/ingress.class: gce
+Events:
+  Type     Reason     Age                  From                     Message
+  ----     ------     ----                 ----                     -------
+  Normal   Sync       19m                  loadbalancer-controller  UrlMap "k8s2-um-20aeohkx-default-cloudarmor-test-0aa8l6ah" created
+  Normal   Sync       19m                  loadbalancer-controller  TargetProxy "k8s2-tp-20aeohkx-default-cloudarmor-test-0aa8l6ah" created
+  Normal   Sync       19m                  loadbalancer-controller  ForwardingRule "k8s2-fr-20aeohkx-default-cloudarmor-test-0aa8l6ah" created
+  Normal   IPChanged  19m                  loadbalancer-controller  IP is now 34.160.135.174
+  Normal   Sync       9m37s (x7 over 20m)  loadbalancer-controller  Scheduled for sync
+```
+
+6. Verify the policy is acting as expected by sending traffic to our Ingress VIP.
+We should expect connection from the IP specified in the policy with path `/whereami` to receive a `whereami` response, whereas connections from different IPs with the same path generating 403s(based on the configured Cloud Armor policy). 
+If step 2 is skipped, requests with non-matching paths should generate 404 responses from the default backend, no matter we are using allowed or blocked IPs. Otherwise, requests using blocked IPs would act depends on the Cloud Armor policy specification(in this example, we should receive 403s), and default 404 response is only received from IPs that allows traffics.
+
+Using allowed IP 34.83.21.159:
+```
+curl 34.160.135.174/whereami
+{"cluster_name":"blueprint-gke-1","gce_instance_id":"6716870310474717637","gce_service_account":"324901199046-compute@developer.gserviceaccount.com","host_header":"34.160.135.174","pod_name":"whereami-59588795bb-7dx2c","pod_name_emoji":"\ud83c\udf24","project_id":"cheungdavid-gke-dev","timestamp":"2023-07-13T23:12:59","zone":"us-west1-a"}
```

Comment:
let's scrub things like the exact service account names etc.

and your name 

chuengdavid-gke-dev

Can you make sure that this is done for the rest of the text.

Reply:
Updated!